src/css/index.css

```
@@ -14,10 +14,8 @@ body {
 }
 
 #app {
-  margin-top: 56px;
   max-width: 600px;
-  margin-left: auto;
-  margin-right: auto;
+  margin: 56px auto 0 auto;
```

left right 값이 같으면 2번째 값만 적어도 동일한 값이 돼요.

```suggestion
  margin: 56px auto 0;
```